keel-clouddriver/src/main/kotlin/com/netflix/spinnaker/keel/clouddriver/CloudDriverService.kt

```diff
@@ -79,6 +80,12 @@ interface CloudDriverService {
     @Header("X-SPINNAKER-USER") user: String = DEFAULT_SERVICE_ACCOUNT
   ): Credential
 
+  @GET("/applications/{application}/loadBalancers")
+  suspend fun loadBalancersForApplication(
+    @Header("X-SPINNAKER-USER") user: String,
+    @Path("application") application: String
+  ): List<AmazonLoadBalancer>
```

Comment:
This CloudDriver endpoint lists all the load balancers for an app. Because there's no individual endpoint for target groups, we need to grab those from any ALBs returned by this list.